src/server/admin/decommission-service/controllers/form.js

```diff
@@ -1,18 +1,32 @@
+import Joi from 'joi'
+
 import { buildOptions } from '~/src/server/common/helpers/options/build-options.js'
 import { fetchRepositories } from '~/src/server/common/helpers/fetch/fetch-repositories.js'
 
 const decommissionFormController = {
+  options: {
+    validate: {
+      query: Joi.object({
+        serviceName: Joi.string().optional()
+      })
+    }
+  },
   handler: async (_request, h) => {
     const { repositories } = await fetchRepositories()
+    const serviceName = _request.query.serviceName ?? ''
```

Comment:
your now using `request`, so the `_` can be removed. The `_` denotes an unused parameter

Reply:
Oh yeah, not sure when the `_` was added